src/blind.rs

```diff
@@ -709,4 +709,39 @@ impl From<ConfidentialTxOutError> for BlindError {
     fn from(from: ConfidentialTxOutError) -> Self {
         BlindError::ConfidentialTxOutError(from)
     }
+}
+
+#[cfg(test)]
+mod tests {
+    use just_imported_for_the_macros::hex::FromHex;
```

Comment:
lol. I'm not gonna hold up the PR for this, but isn't there another more direct way to refer to this `hex` module?